app/src/main/res/values/strings.xml

```diff
@@ -238,7 +238,7 @@
     <string name="fingerprint">Fingerprint</string>
     <string name="fingerprint_enable_title">Fingerprint scanning</string>
     <string name="fingerprint_enable_summary">Lets you scan your fingerprint to open your database.</string>
-    <string name="fingerprint_delete_all_title">Delete encryption keys</string>
+    <string name="fingerprint_delete_all_title">Remove App Security</string>
```

Comment:
Remove App Security ? Abolutely not! It's a subsection of fingerprint, and only delete the encryption keys from the fingerprint keystore
Why are you capitalize each word ? System app don't do that